corehq/apps/zapier/views.py

```diff
@@ -86,3 +90,71 @@ def post(self, request, *args, **kwargs):
             return HttpResponse(status=400)
         delete_subscription_with_url(url)
         return HttpResponse('OK')
+
+
+class ZapierCreateCase(View):
+
+    urlname = 'zapier_create_case'
+
+    @method_decorator(csrf_exempt)
+    def dispatch(self, *args, **kwargs):
+        return super(ZapierCreateCase, self).dispatch(*args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        domain = request.GET.get('domain')
+        case_type = request.GET.get('case_type')
+        owner_id = request.GET.get('user_id')
```

Comment:
I think this is confusing. Can you change `user_id` to `owner_id` in Zapier so this is consistent?